src/radha/CodingTest1/Program2.java

```diff
@@ -0,0 +1,49 @@
+/*
+ -------------------------------------------------------------------
+Program 2 : 
+WAP to provide the difference between a Younger and an elder member of the family. Age is provided in the form of an array.
+
+Sample Input: 7 12 56 62 25
+
+Sample Output: 55
+ */
+
+package radha.CodingTest1;
+
+public class Program2 {
+	
+	int getMaxAge(int[] num) {
+		int maxAge = num[0];
+		for(int index=1; index < num.length;index++) {
+			if(maxAge < num[index]) 
+				maxAge=num[index];
+		}
+		return maxAge;
+	}
+	
+	int getMinAge(int[] num) {
```

Comment:
This method should merge with above ideally, and you can get both value at same place